[2021-07-23 17:07:23,282] {taskinstance.py:896} INFO - Dependencies all met for <TaskInstance: hello_world.load_to_db 2021-07-23T14:07:17.686307+00:00 [queued]>
[2021-07-23 17:07:23,298] {taskinstance.py:896} INFO - Dependencies all met for <TaskInstance: hello_world.load_to_db 2021-07-23T14:07:17.686307+00:00 [queued]>
[2021-07-23 17:07:23,298] {taskinstance.py:1087} INFO - 
--------------------------------------------------------------------------------
[2021-07-23 17:07:23,299] {taskinstance.py:1088} INFO - Starting attempt 1 of 2
[2021-07-23 17:07:23,299] {taskinstance.py:1089} INFO - 
--------------------------------------------------------------------------------
[2021-07-23 17:07:23,322] {taskinstance.py:1107} INFO - Executing <Task(BashOperator): load_to_db> on 2021-07-23T14:07:17.686307+00:00
[2021-07-23 17:07:23,324] {standard_task_runner.py:52} INFO - Started process 22114 to run task
[2021-07-23 17:07:23,332] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'hello_world', 'load_to_db', '2021-07-23T14:07:17.686307+00:00', '--job-id', '53', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/simple_dag.py', '--cfg-path', '/tmp/tmp4k0zfe99', '--error-file', '/tmp/tmppowtr1hc']
[2021-07-23 17:07:23,333] {standard_task_runner.py:77} INFO - Job 53: Subtask load_to_db
[2021-07-23 17:07:23,387] {logging_mixin.py:104} INFO - Running <TaskInstance: hello_world.load_to_db 2021-07-23T14:07:17.686307+00:00 [running]> on host DESKTOP-JHC8U0L.localdomain
[2021-07-23 17:07:23,464] {taskinstance.py:1300} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=airflow@example.com
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=hello_world
AIRFLOW_CTX_TASK_ID=load_to_db
AIRFLOW_CTX_EXECUTION_DATE=2021-07-23T14:07:17.686307+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2021-07-23T14:07:17.686307+00:00
[2021-07-23 17:07:23,467] {subprocess.py:52} INFO - Tmp dir root location: 
 /tmp
[2021-07-23 17:07:23,468] {subprocess.py:63} INFO - Running command: ['bash', '-c', 'python3 /home/dennismutia/vouchers/load_to_db.py']
[2021-07-23 17:07:23,476] {subprocess.py:74} INFO - Output:
[2021-07-23 17:07:24,734] {subprocess.py:82} INFO - Command exited with return code 0
[2021-07-23 17:07:24,781] {taskinstance.py:1204} INFO - Marking task as SUCCESS. dag_id=hello_world, task_id=load_to_db, execution_date=20210723T140717, start_date=20210723T140723, end_date=20210723T140724
[2021-07-23 17:07:24,841] {taskinstance.py:1265} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2021-07-23 17:07:24,867] {local_task_job.py:149} INFO - Task exited with return code 0
